packages/contracts/hardhat.config.ts

```diff
@@ -42,20 +42,13 @@ for (const network of Object.keys(hardhatNetworks) as SupportedNetworks[]) {
 
 // Extend HardhatRuntimeEnvironment
 extendEnvironment(hre => {
-  const aragonPluginRepos: AragonPluginRepos = {
-    AddresslistVotingRepoProxy: '',
-    TokenVotingRepoProxy: '',
-    AdminRepoProxy: '',
-    MultisigRepoProxy: '',
-  };
   const testingFork: TestingFork = {
     network: '',
     osxVersion: '',
     activeContracts: {},
   };
-  hre.aragonPluginRepos = aragonPluginRepos;
   hre.aragonToVerifyContracts = [];
-  hre.managementDAOMultisigPluginAddress = '';
+  hre.managementDAOMultisigPluginAddress = ''; // TODO
```

Comment:
```suggestion
  hre.managementDAOMultisigPluginAddress = ''; // TODO
```
```suggestion
  hre.managementDAOMultisigPluginAddress = ''; // TODO This must be removed after the deploy script got refactored (see https://github.com/aragon/osx/pull/582)
```

Reply:
Remember to take care of removing these vars in https://github.com/aragon/osx/pull/582